md-dao/src/main/java/com/wipro/ats/bdre/md/dao/ProcessLogDAO.java

```diff
@@ -210,9 +211,9 @@ public ProcessLogInfo getLastValue(ProcessLogInfo processLogInfo) {
 
                 }
                 }else {
-                if (processList.size() != 0) {
+                if (!processList.isEmpty()) {
                     for (Integer processId : processList) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)